source/reporting/Report database.rst

```diff
@@ -0,0 +1,248 @@
+
+When you send a tag with an order, two tags are currently created in the report layer. One with type Subscriber and one with type SubscriberAccount. Referenceid then correspond henholdsvi s SubscriberId and SubscriberAccount . Thus you can search for all subscribers who have a relationship with for example ShopID = 458. The vast majority of tables in the report layer have a connection to subscriberide and can then be grouped / filtered in relation to Tag.
+ 
+
+[economy].[AccountTransaction]
+----------------------------
+Saldo transactions can, among other things, be used for purchases from agreements. For example, if you agree on a payment claim, which you will later buy out of, then this table will be used.
+ 
+
+[economy].[Invoices]
+------------------
+Invoice details.
+
+ 
+[economy].[PaymentDemandAllowances]
+---------------------------------
+Contains adjustments that occur when you pay too much or too little.
+ 
+
+[economy].[PaymentDemandCharges]
+------------------------------
+Balance sheet transactions that are included in payment claims.
+ 
+
+[economy].[PaymentDemandDetails]
+--------------------------------
+Information about payment requirements, connected to the subscription (Subscription) and what is the next subscriptionId.
+
+Payment claims are formed before the subscription is formed. Therefore, subscriptionid is for the previous subscription, while NextsubscriptionId is the subscription in question. When a subscription is renewed, NextSubscriptionId is applied to the correct payment claim.
+ 
+[economy].[PaymentDemandFees]
+---------------------------
+Invoice fee.
+
+
+[economy].[PaymentDemands]
+--------------------------
+Invoicing of customers produces a demand for payment, and PaymentDemands contains customers' invoices, with amounts, time for when they were issued and when the demand is due. Link to ledger is displayed.
+
+Payment Demand is the basis for payment claims that the system generates, and it is on the basis of these that an invoice is formed. Examples of using payment requirements k an be reporting outstanding guilty amount or income broken down by month of the year. Each claim points to the subscription period that is the basis for the claim. It is important here to keep in mind that the requirement is formed before the period is formed, so the pointer is therefore always backwards in time.  
+
+For orders, there will be a pointer to the order that led to the claim.
+ 
+
+[economy].[SubscriberAmounts]
+-----------------------------
+Information on all amounts paid for a subscriber, divided into different subscriptions.
+
+ 
+
+[economy].[SubscriberLedgers]
+-----------------------------
+Displays the customer ledger and the types of financial transactions that have been performed for the subscriber.
```

Comment:
This is all ledger-entries for all subscribers.

Reply:
Updated